portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeFooterExtras.vm

```diff
@@ -293,7 +293,14 @@
                 //these are global variables...
                 var sakaiPresenceTimeDelay = ${sitePages.sakaiPresenceTimeDelay};
                 var sakaiPresenceFragment = "${sitePages.pageNavPresenceUrl}?output_fragment=yes&auto=true";
-                var sakaiLastPresenceTimeOut = setTimeout('updatePresence()', sakaiPresenceTimeDelay); 
+                var sakaiLastPresenceTimeOut = setTimeout('firstRun()', sakaiPresenceTimeDelay);
+                var firstRun = function() {
```

Comment:
I would change the name of this function. It's always run, just with a possible delay first time round.

Reply:
Changed from 'firstRun' to 'delayedUpdatePresence'